projects/microphysics/scripts/piggy-back.py

```diff
@@ -144,9 +167,39 @@ def time_dependent_dataset(skills):
     return wandb.Table(dataframe=df)
 
 
-metrics["skill"] = time_dependent_dataset(skills)
-metrics["skill_time"] = time_dependent_dataset(skills_time)
-wandb.log(metrics)
+def log_lat_vs_p_skill(field):
+    """Returns a function that will compute the lat vs pressure skill of a field"""
+
+    def func(ds):
+        return {
+            f"lat_p_skill/{field}": wandb.Plotly(
+                plot_cloud_skill_zonal(ds, field, "mean")
+            )
+        }
+
+    return func
+
+
+log_functions = [
```

Comment:
I refactored this script to build up a list of functions with the same interface rather than calling them directly...this moves towards a "registry" style approach.